sdk/ml/azure-ai-ml/azure/ai/ml/entities/_job/_input_output_helpers.py

```diff
@@ -358,12 +368,16 @@ def from_rest_data_outputs(outputs: Dict[str, RestJobOutput]) -> Dict[str, Outpu
         # deal with invalid output type submitted by feb api
         # todo: backend help convert node level input/output type
         normalize_job_input_output_type(output_value)
-
+        if getattr(output_value, "pathOnCompute", None) is not None:
```

Comment:
why this need to use getattr unlike other fields?

Reply:
Current version of JobOutput model does not have the attribute of "pathOnCompute" because the changed mfe.json has not released yet.  1. we check the JobOutput has attribute of "pathOnCompute" or not 2. If it does not have its attribute as "pathOnCompute", we will parse None to Output class.